cpp/src/dlpack/dlpack.cpp

```diff
@@ -51,7 +51,10 @@ data_type DLDataType_to_data_type(DLDataType type)
     }
   } else if (type.code == kDLUInt) {
     switch (type.bits) {
-      case 8: return data_type(BOOL8);
```

Comment:
Where does `BOOL8` go now? is it removed?

Reply:
cupy throws an error when DLPack-ing boolean arrays so we decided we should too
https://github.com/rapidsai/cudf/issues/5353#issuecomment-638457572